pyerrors/correlators.py

```diff
@@ -1081,7 +1066,7 @@ def _sort_vectors(vec_set, ts):
 
 
 def _GEVP_solver(Gt, G0):  # Just so normalization an sorting does not need to be repeated. Here we could later put in some checks
-    sp_val, sp_vecs = scipy.linalg.eig(Gt, G0)
+    sp_val, sp_vecs = scipy.linalg.eigh(Gt, G0)
```

Comment:
By now I am not sure anymore whether `eigh` actually checks if the matrix is positive semi definite. Do we need an additional safeguard here or not?

Reply:
Yes, it does. "If eigenvalue computation does not converge, an error occurred, or b matrix is not definite positive. Note that if input matrices are not symmetric or Hermitian, no error will be reported but results will be wrong."